enterprise/views.py

```diff
@@ -958,8 +958,9 @@ def get_available_course_modes(self, request, course_run_id, enterprise_catalog)
             modes.sort(key=lambda course_mode: enterprise_catalog.enabled_course_modes.index(course_mode['slug']))
             if not modes:
                 LOGGER.info(
-                    'No matching course modes found for course run {course_run_id} in '
-                    'EnterpriseCustomerCatalog [{enterprise_catalog_uuid}]'.format(
+                    '[Enterprise Enrollment] Matching course modes were not found in EnterpriseCustomerCatalog. '
+                    'CourseRun: {course_run_id}, '
```

Comment:
Same here about course run id

Reply:
Course and CourseRun are logged separately elsewhere in the file also. Hence following the same convention.